commit 91b66a6b402b790f3c8cebb0420ef549744ee197
Author: Sergey V. Lobanov <sergey@lobanov.in>
Date:   Mon Jan 3 19:25:45 2022 +0300

    add compatibility for wolfssl >= 5.0
    
    NTRU support has been removed in wolfssl 5.0 so it is required to
    mask NTRU specific code if wolfssl >= 5.0

--- a/src/ssl/openssl.c
+++ b/src/ssl/openssl.c
@@ -314,7 +314,9 @@ static bool handle_wolfssl_asn_error(voi
     case ASN_SIG_HASH_E:
     case ASN_SIG_KEY_E:
     case ASN_DH_KEY_E:
+#if LIBWOLFSSL_VERSION_HEX < 0x05000000
     case ASN_NTRU_KEY_E:
+#endif
     case ASN_CRIT_EXT_E:
     case ASN_ALT_NAME_E:
     case ASN_NO_PEM_HEADER:
